content/html/reference/elements/acroynm/acronym.md

```diff
@@ -0,0 +1,44 @@
+---
+recipe: html-element
+title: '<acronym>'
+mdn_url: /en-US/docs/Web/HTML/Element/acronym
+browser_compatibility: html.elements.acronym
+attributes:
+  global: /content/html/global_attributes
+examples:
+  - examples/example
+specifications: non-standard
+---
+
+## Short description
+
+The HTML Acronym Element (`<acronym>`) allows authors to clearly indicate a sequence of characters that compose an acronym or abbreviation for a word.
+
+## Overview
+
+This element has been removed in HTML5 and shouldn't be used anymore. Instead web developers should use the [`<abbr>`](/en-US/docs/Web/HTML/Element/abbr) element.
+
+## DOM Interface
```

Comment:
This is getting outside the scope of "migration", but this is an example of a section that I'd just delete. We will capture the DOM interface as structured content, and noone cares about what happened in Gecko 1.9.2.

Reply:
Done.